incubator/cassandra/values.yaml

```diff
@@ -0,0 +1,44 @@
+## Cassandra image version
+## ref: https://hub.docker.com/r/library/cassandra/
+image:
+  repo: "cassandra"
+  tag: "latest"
+  pullPolicy: IfNotPresent
+
+## Specify a service type
+## ref: http://kubernetes.io/docs/user-guide/services/
+service:
+  type: ClusterIP
+
+## Persist data to a persitent volume
+persistence:
+  enabled: true
+  storageClass: generic
+  accessMode: ReadWriteOnce
+  size: 100Gi
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources:
+  requests:
+    memory: 10Gi
```

Comment:
This chart should be ready to use in small production environment. The minimum recommend HEAP size of Cassandra is 8 GB. So I put a resource memory to 10Gi and `max_heap_size: 8192M`.